ChatSecure/Classes/Controllers/XMPP/OTRXMPPRoomManager.m

```diff
@@ -148,6 +154,7 @@ - (void)inviteBuddies:(NSArray<NSString *> *)buddyUniqueIds toRoom:(XMPPRoom *)r
     }];
     [buddies enumerateObjectsUsingBlock:^(OTRXMPPBuddy * _Nonnull buddy, NSUInteger idx, BOOL * _Nonnull stop) {
         [self inviteUser:buddy.bareJID toRoom:room.roomJID withMessage:nil];
+        [self setUser:buddy.bareJID affiliation:@"admin" inRoom:room.roomJID];
```

Comment:
I agree, I'm not sure if this is a good default for upstream because it may lead to unexpected behavior for more advanced XMPP users. We already have complaints that we are setting room parameters for every room we join, which I'm not sure if it's been fixed. However perhaps it could be a default if it was at least configurable.

I think there is still a use case for Zom for having a toggle to not make everyone an admin, for example a very large group chat with potentially less trusted invited members. I wonder if we could have an button on the invite view to push a "Group Options" view that would allow for additional configuration flexibility in the future. For now it could have the single option:

* **Trust Invited Contacts** subtitle: Invited members will be made room administrators. When creating a large room with less untrusted contacts, you may want to disable this. default: ON

The room options could be a serializable object (MTLModel subclass I guess) with a single boolean option `makeInviteesAdmin`. Then we could store the room options on the OTRXMPPRoom object as  `roomOptions`, and use that for this logic.

Then the below logic could look like:

```objc
OTRXMPPRoom *xmppRoom = ...
if (xmppRoom.roomOptions.makeInviteesAdmin) {
    [self setUser:buddy.bareJID affiliation:@"admin" inRoom:room.roomJID];
}
```

cc @cstiens

Reply:
I agree, making all group members admin somehow feels... wrong. Let's discuss today.